packages/utils/src/object/mapKeys/mapKeys.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { mapKeys } from '.';
+
+describe('mapKeys', () => {
+  it('should map object keys by appending values to keys', () => {
+    const obj = { a: 1, b: 2 } as const;
+    const iteratee = ({
+      key,
+      value,
+    }: {
+      key: string | number;
+      value: number;
+    }): string => `${key}${value}`;
+    const expected = { a1: 1, b2: 2 } as const;
+
+    const result = mapKeys(obj, iteratee);
+
+    expect(result).toEqual(expected);
+  });
```

Comment:
```suggestion
  it('should map object keys by appending values to keys', () => {
    const obj = { a: 1, b: 2 } as const;

    const expected = { a1: 1, b2: 2 } as const;

    const result = mapKeys(obj, ({ key, value }) => `${key}${value}`);

    expect(result).toEqual(expected);
  });
```

원활하게 타입 추론을 되는지 확인하기 위해 테스트 코드 작성 시 iteratee를 추출해서 인자로 넣기 보다 그냥 위와 같이 작성하는게 이해하는데 더  원활할 것 같습니다!!

<img width="628" alt="스크린샷 2024-08-02 오전 1 07 51" src="https://github.com/user-attachments/assets/f4162908-7a48-40d7-a3a2-76f88dee9fbe">

Reply:
수정했습니다!